javascript/organizing-js/oop-concepts.md

```diff
@@ -79,3 +79,4 @@ This section contains helpful links to other content. It isn't required, so cons
 
 * The best book we've ever read on the subject of loose coupling is [Practical Object-Oriented Design In Ruby](http://www.poodr.com/). Unfortunately, it is not free.. and not JavaScript. We feel confident in recommending it anyway. If you don't know Ruby, it is a clear enough language that you don't really need to learn it to follow the examples and the content of the book is sincerely fantastic.
 
+* A nice summary of [OOP's most commun principles using JavaScript](https://medium.com/better-programming/object-oriented-programming-in-javascript-b3bda28d3e81)
```

Comment:
Please ensure proper spelling in your PRs

"commun" -> "common"

Reply:
will pay more attention. sorry